en/gps_compass/rtk_gps_cuav_c-rtk2.md

```diff
@@ -0,0 +1,115 @@
+# CUAV C-RTK2
+
+The [CUAV C-RTK2 receiver](http://doc.cuav.net/gps/c-rtk/en/) is a high-performance PPK/RTK positioning module created by CUAV for professional applications such as drone aerial survey (Mapping).
+It has a high-precision IMU and positioning module.
+It can reduce more than 80% of control points.
+it can also be used as an RTK receiver.
```

Comment:
Above you make the point "is a high-performance PPK/RTK positioning module". How is "it can also be used as an RTK receiver." different than this? 
I.e. it sounds like you are saying something twice, but you might be intending some distinction between the two things.

Does my question make sense?